cargoTracking.md

```diff
@@ -0,0 +1,123 @@
+# Gönderi Takip Apisi
```

Comment:
Api'si yerine servisi desek nasıl olur?

Reply:
Api ama bu :)

Reply:
doğru :D türkçe eklemeler olunca tuhaf oluyor alternatif aradım onun için ahaha